help.go

```diff
@@ -101,6 +101,10 @@ func (e *Executor) ListTaskNames(allTasks bool) {
 	// sort and print all task names
 	sort.Strings(s)
 	for _, t := range s {
-		fmt.Fprintln(w, t)
+		_, err := fmt.Fprintln(w, t)
+		if err != nil {
+			log.Fatal(err)
+			return
+		}
```

Comment:
I recommend rolling back any changes on this file as it's not needed for this patch and will conflict with other PRs where we're editing it, like #874.